### PR TITLE
adding flow device definition

### PR DESCRIPTION
**Pull request by thezackm**
### Relevant information

adding definition for flow devices from Kentik

### Checklist

* [x] I've read the guidelines and understand the acceptance criteria.
* [x] The value of the attribute marked as `identifier` will be unique and valid. 
* [x] I've confirmed that my entity type wasn't already defined. If it is I'm providing an
 explanation above.

### CODE REVIEW

**Review by srvaroa — APPROVED** [review]
Thanks!

**Review by IreneMLC — APPROVED** [review]
Looks good, thanks!

File: definitions/ext-flow_device/summary_metrics.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    select: uniqueCount(ip_address)
+    from: Metric
+    where: "metricName LIKE 'kentik.rollup.bytes.%'"
+    eventId: entity.guid
+
+conversations:
+  title: Conversations
+  unit: COUNT
+  query:
+    select: uniqueCount(tuple(ip_address,port))
```
Comment (naxhh) [inline definitions/ext-flow_device/summary_metrics.yml:14]: unsure if this will work out of the box in summaries. we should check

Comment (thezackm) [reply definitions/ext-flow_device/summary_metrics.yml:14]: it seems to be working right now in the [ext-vpc_network](https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/b972d23d8ab26cec93dfc77a0ea5ae8aacbeb3fd/definitions/ext-vpc_network/summary_metrics.yml#L9-L16) entities that are live

**Pull request merged**